src/questions/javascript-questions.md

```diff
@@ -21,6 +21,7 @@ permalink: /questions/javascript-questions/index.html
 * Explain `Function.prototype.bind`.
 * What's the difference between feature detection, feature inference, and using the UA string?
 * Explain "hoisting".
+* What are truthy and falsy values in Javascript?
```

Comment:
What is type coercion? What are common pitfalls of relying on type coercion in JavaScript code?

Reply:
This has been fixed